### PR TITLE
Create aex_SSS.md

### PR DESCRIPTION
**Pull request by nikita-fuchs**
This is the first draft, ready for first reviews and discussions. Thank you for your consideration.

### CODE REVIEW

File: AEXS/aex-6.md [hunk 1]
```diff
@@ -0,0 +1,682 @@
+# AEX 6: Strategic Semantic Syntax: Contract Interaction Wrapper 
+<style>
+.markdown-body { max-width: 80% !important; }
+</style>
+```
+AEX: 6
+Title: Strategic Semantic Syntax Part 1: Contract Interaction Wrapper
+Author: Nikita Fuchs GitHub: @nikita-fuchs
```
Comment (knarz) [inline AEXS/aex-6.md:8]: `Author: Nikita Fuchs (@nikita-fuchs)`

File: AEXS/aex-6.md [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+
+
+## Abstract
+
+Currently, the SDK is not aware of any characteristics of any smart contract or any of its functions. Any information is provided through hard coding of names, type info and explicit decoding instructions. Additionally, a lack of a formalized strategic concept in the development of the SDK syntax makes æepp development tedious and hindered by seemingly arbitrary breaking changes through adjustments and every new feature.
+
+
+## Motivation
+
+The team did an amazing job so far implementing SDK functionalities in an evolutionary way. People from different backgrounds did the best imaginable job to add functionalities to the SDK subsequently and in a way that made the best sense to them. We can be very thankful to have these many talented JS devs in our team! Now, as the project matures, we face at least two challenges:
```
Comment (knarz) [inline AEXS/aex-6.md:31]: While this is true, most of this seems like fluff and probably doesn't need to go into this proposal text :)

File: AEXS/aex-6.md [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+
+Currently, the SDK is not aware of any characteristics of any smart contract or any of its functions. Any information is provided through hard coding of names, type info and explicit decoding instructions. Additionally, a lack of a formalized strategic concept in the development of the SDK syntax makes æepp development tedious and hindered by seemingly arbitrary breaking changes through adjustments and every new feature.
+
+
+## Motivation
+
+The team did an amazing job so far implementing SDK functionalities in an evolutionary way. People from different backgrounds did the best imaginable job to add functionalities to the SDK subsequently and in a way that made the best sense to them. We can be very thankful to have these many talented JS devs in our team! Now, as the project matures, we face at least two challenges: 
+- The learnings we make and additions we come up with will amost always lead to breaking changes
+
+- So will implementing further AEX, too, providing advanced functionalities like device-specific behaviour, deep-linking, etc.
```
Comment (knarz) [inline AEXS/aex-6.md:34]: Aren't these two points basically the same single challenge, ie there will be breaking changes in the future?

File: AEXS/aex-6.md [hunk 4]
```diff
@@ -0,0 +1,682 @@
...
+
+Therefor I would like to propose a Strategic Semantic Syntax, that is supposed to wrap everything neccessary to interact with contracts (and later state channels, oracles, and events) in function calls that take only the bare minimum of *semantic* information and leaves the heavy-lifting to the SDK and Framework.
```
Comment (knarz) [inline AEXS/aex-6.md:36]: Therefore

**Comment by thepiwo** [comment]
will this still be adjusted/accepted? (cc @noandrea )

**Comment by marc0olo** [comment]
@nikita-fuchs @thepiwo @mradkov can we close that? not really sure what this is about. I don't think it's worth to put some time in that to be honest

**Pull request closed without merging**